arrange_schedule.py

```diff
@@ -1035,7 +1035,124 @@ def check_event_exist_or_insert(event):
         with open(receive_msg["text_system_dir"],"w") as fp:
             print(json.dumps(text_file),file=fp)
 
-#crawler google drive image
+def find_drive_data_type():
+    return_msg = {}
+    with DataTypeDao() as dataTypeDao:
+        typeId = dataTypeDao.getTypeId('google_drive_image')
+    if typeId != None:
+        return_msg['data_type'] = int(typeId)
+        return_msg['result'] = 'success'
+        return return_msg
+    else:
+        return_msg['error'] = "no google_drive_image data type"
+        return_msg['result'] = 'fail'
+        return return_msg
+
+def search_google_drive_folder(service):
+    g_sql = "(name='1day' or name='3day' or name='7day' or name='14day')"
+    results = service.files().list(
+        q="mimeType='application/vnd.google-apps.folder' and trashed=false and " + g_sql, 
+        fields="nextPageToken, files(id, name)").execute()
+    items = results.get('files', [])
+    return items
+
+def search_google_drive_file(service):
+    #set time
+    now_time = time.time()
+    start_time = time.strftime("%Y-%m-%dT%H:%M:%S+08:00", time.localtime(now_time-43200))
+    results = service.files().list(
+        q="modifiedTime > '" + start_time + "' and mimeType contains 'image/'", 
```

Comment:
Maybe line 1064 could change to
`"modifiedTime > '{start_time}' and mimeType contains 'image/'".format(start_time=start_time)`